src/main/java/com/alkemy/ong/model/request/CreateMemberRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.alkemy.ong.model.request;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateMemberRequest {
+
+  @NotBlank(message = "Name is mandatory")
```

Comment:
MINOR: para ser consistente con los mensajes que existen en el proyecto lo modificaría a: "Name cannot be null or empty."